docs/install/install-server/install-server.inc

```diff
@@ -118,10 +179,12 @@ Install the software
 .. warning::
     When you use Inmanta modules that depend on python libraries with native code, python headers and a working compiler are required as well.
 
+.. _configure-server:
+
 Configure server
 ################
 This guide goes through the steps to set up an Inmanta service orchestrator server. This guide assumes a RHEL (7 or 8) or CentOS
-(7 or 8) server is used. The rpm packages install the server configuration file in `/etc/inmanta/inmanta.cfg`.
+(7 or 8) server is used. The rpm packages install the server configuration file in `/etc/inmanta/inmanta.cfg`. We will add a detailed part for Debian and its derivatives in the near future but this section will guide you to the right direction. The only difference is some configuration paths are slightly different in Debian.
```

Comment:
I don't think we need to add this information to the documentation. It also doesn't add much value, in my opinion.